### PR TITLE
tests for books model

### PR DESCRIPTION
**Pull request by MaaooVaass**
## summary
tests for books model
## Cards
https://trello.com/c/z381r4Zx/21-crear-tests-para-el-modelo-de-book

### CODE REVIEW

File: app/models/book.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class Book < ApplicationRecord
+  validates :genre, :author, :image, :title, :publisher, :year, presence: true
+end
```
Comment (fedeest) [inline app/models/book.rb]: Didn't you include this already on some other PR?

**Review by fedeest — CHANGES_REQUESTED** [review]

File: Gemfile [hunk 2]
```diff
@@ -33,6 +33,7 @@ gem 'rubocop', '~> 0.60.0', require: false
 gem 'omniauth'
 # Use Capistrano for deployment
 # gem 'capistrano-rails', group: :development
+gem 'faker', :git => 'https://github.com/stympy/faker.git', :branch => 'master'
```
Comment (lmasello) [inline Gemfile]: It's better to specify the gem version.
Example:
` gem 'faker', '~> 1.9.1' `

File: spec/book_spec.rb [hunk 3]
```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+describe Book, type: :model do
+  it do
```
Comment (lmasello) [inline spec/book_spec.rb]: We can put each it in just one line :)
` it { should validate_presence_of(:genre) }`

We can also use is_expected.to:
`it { is_expected.to validate_presence_of(:genre) }`

File: spec/book_spec.rb [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    should validate_presence_of(:title)
+  end
+  it do
+    should validate_presence_of(:publisher)
+  end
+  it do
+    should validate_presence_of(:year)
+  end
+  subject(:book) do
```
Comment (lmasello) [inline spec/book_spec.rb:4]: It's also better to write subject and lets before the 'it's

**Review by lmasello — CHANGES_REQUESTED** [review]

File: spec/book_spec.rb (hunk 3 above)
Comment (lmasello) [reply spec/book_spec.rb]: https://github.com/thoughtbot/shoulda-matchers#should-vs-is_expectedto

**Review by lmasello — APPROVED** [review]

**Review by fedeest — APPROVED** [review]

**Pull request merged**